src/lib/discord.js

```diff
@@ -0,0 +1,18 @@
+const sendMessage = (user, repoUrl) => {
+  const request = new XMLHttpRequest();
```

Comment:
using this request because it was in the discord.js docs. I don't have a preference. Does anyone else have a preference?

Reply:
my personal is `const https = require('https');` for server side things, a little bit verbose tho. in client context XHR is just fine!

Reply:
We will leave it as-is for now, since it is the trusted discord solution. But open to future contribs on this.